client/remote.go

```diff
@@ -188,3 +189,14 @@ func (remote *RemoteVPS) getDaemonAPIToken(session SSHSession, daemonVersion str
 	// There may be a newline, remove it.
 	return strings.TrimSuffix(stdout.String(), "\n"), nil
 }
+
+func (remote *RemoteVPS) SetRemoteProperty(name string, value string) bool {
```

Comment:
Not sure if this function is needed - see other comment on `config.SetProperty()`